config/routes.rb

```diff
@@ -9,7 +9,7 @@
   end
 
   namespace :admin do
-    resources :rebate_forms, only: %i[show index update destroy]
+    resources :rebate_forms, only: %i[show index update destroy edit]
```

Comment:
this can be just 
`resources :rebate_forms`
because we have all the methods on there now.